WcaOnRails/app/jobs/timed_application_job.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class TimedApplicationJob < ApplicationJob
+  class << self
+    def start_timestamp
+      Timestamp.find_or_create_by(name: "#{self.name.underscore}_start")
```

Comment:
Doesn't matter here, as we don't have validations, but sure I can add the `!` =)